oxidation/libparsec_python/src/addrs.rs

```diff
@@ -97,7 +97,7 @@ impl BackendAddr {
 
 #[pyclass]
 #[derive(PartialEq, Eq, Clone)]
-pub(crate) struct BackendOrganizationAddr(parsec_api_types::BackendOrganizationAddr);
+pub(crate) struct BackendOrganizationAddr(pub parsec_api_types::BackendOrganizationAddr);
```

Comment:
```suggestion
pub(crate) struct BackendOrganizationAddr(pub(crate) parsec_api_types::BackendOrganizationAddr);
```

shouldn't it be `pub(crate)` ? since the struct is only visible to the crate ?

Reply:
useless, because you can't access it anyway